superset-frontend/stylesheets/superset.less

```diff
@@ -590,3 +590,9 @@ hr {
   background-image: url('../images/icons/error_solid_small_red.svg') !important;
   background-position: -2px center !important;
 }
+
+.Select__value-container--is-multi .Select__input {
+  input {
+    width: 100% !important;
```

Comment:
It doesn't look like react-select exposes this.

Reply:
I tested locally and it seems to work better without this line. Plus this looks very very hacky... I'm not sure it's worth it. 


The diff that worked for me:

```diff
diff --git a/superset-frontend/src/components/Select/styles.tsx b/superset-frontend/src/components/Select/styles.tsx
index f8c2e8825..ed476ffde 100644
--- a/superset-frontend/src/components/Select/styles.tsx
+++ b/superset-frontend/src/components/Select/styles.tsx
@@ -312,6 +312,9 @@ export const DEFAULT_COMPONENTS: SelectComponentsType = {
       <Input
         {...props}
         placeholder={isMultiWithValue ? placeholder : undefined}
+        // disable Chrome's address autocomplete
+        // @ts-ignore
+        autocomplete="chrome-off"
         css={getStyles('input', props)}
       />
     );
diff --git a/superset-frontend/stylesheets/superset.less b/superset-frontend/stylesheets/superset.less
index adb56ea11..cdd259b6d 100644
--- a/superset-frontend/stylesheets/superset.less
+++ b/superset-frontend/stylesheets/superset.less
@@ -590,9 +590,3 @@ hr {
   background-image: url('../images/icons/error_solid_small_red.svg') !important;
   background-position: -2px center !important;
 }
-
-.Select__value-container--is-multi .Select__input {
-  input {
-    width: 100% !important;
-  }
-}
```